### PR TITLE
Generated project helper package.json.

### PR DESCRIPTION
**Pull request by Snider**
As NPM is a Wails dependancy adding a simple package.json to a generated derivative project will help colaberators on those projects.

Some of these people will be designers and web devs with zero GoLang experience.

I have added a postinstall hook so that `npm install` at a derivative projects root will install Wails & any dependancies found in fontend/package.json

This means a prerequesite for derivative projects is simly "Install GoLang, NodeJS and `run npm install`"

### CODE REVIEW

**Comment by leaanthony** [comment]
Thanks for raising this. How is the postinstall hook triggered?

**Comment by Snider** [comment]
it's run every time `npm install` is run as a post-process hook.

File: v2/cmd/wails/internal/commands/generate/template/base/package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "private": true,
+  "scripts": {
+    "postinstall": "npm run setup && cd frontend && npm install",
+    "build": "wails build --clean",
+    "build:macos": "npm run build -- --platform darwin/universal",
+    "build:macos-arm": "npm run build -- --platform darwin/arm64",
+    "build:macos-intel": "npm run build -- --platform darwin",
+    "build:windows": "npm run build -- --platform windows/amd64",
+    "setup": "go install github.com/wailsapp/wails/v2/cmd/wails@v2.0.0-beta.20"
```
Comment (leaanthony) [inline v2/cmd/wails/internal/commands/generate/template/base/package.json:10]: Could we change this to `@latest`?

**Review by leaanthony — CHANGES_REQUESTED** [review]

**Comment by Snider** [comment]
ugg, yes. i was not on master when i pressed that, darn.

**Pull request closed without merging**